mmdet/models/mask_heads/fcn_mask_head.py

```diff
@@ -20,27 +21,30 @@ def __init__(self,
                  in_channels=256,
                  conv_kernel_size=3,
                  conv_out_channels=256,
-                 upsample_method='deconv',
-                 upsample_ratio=2,
                  num_classes=81,
                  class_agnostic=False,
+                 upsample_cfg=dict(type='deconv', upsample_ratio=2),
                  conv_cfg=None,
                  norm_cfg=None,
                  loss_mask=dict(
                      type='CrossEntropyLoss', use_mask=True, loss_weight=1.0)):
         super(FCNMaskHead, self).__init__()
-        if upsample_method not in [None, 'deconv', 'nearest', 'bilinear']:
+        self.upsample_cfg = upsample_cfg.copy()
```

Comment:
`upsample_cfg` may not be necessary as an attribute, since we will set `self.upsample_method`, and other fields can be obtained in `self.upsample`.